CONTRIBUTING.md

```diff
@@ -1,89 +1,41 @@
 # Contribution Guideline
 
 Starknet Foundry is under active development and is open for contributions!
-Want to get started?
-Grab any [issue](https://github.com/foundry-rs/starknet-foundry/issues) labeled with `good-first-issue`!
-Need some guidance?
 
-Reach out to other developers on [Telegram](https://t.me/+d8ULaPxeRqlhMDNk) or open
-a [GitHub discussion](https://github.com/foundry-rs/starknet-foundry/discussions)!
-
-### Environment setup
-
-See [development guide](https://foundry-rs.github.io/starknet-foundry/development/environment-setup.html) in Starknet
-Foundry book for environment setup.
-
-### Running Tests and Checks
-
-> ⚠️ Make sure you run `./scripts/install_devnet.sh`
-> and then set [Scarb](https://docs.swmansion.com/scarb/) version
-> [compatible](https://github.com/foundry-rs/starknet-foundry/releases) with both `snforge` and `sncast`
-> after setting up the development environment, otherwise the tests will fail.
-
-Before creating a contribution, make sure your code passes the following checks
-
-```shell
-$ cargo test
-$ cargo fmt --check
-$ cargo lint
-$ typos
-```
+## Opening an issue 
 
-Otherwise, it won't be possible to merge your contribution.
+If you think something doesn't work or something is missing please open an issue! This way we can address this problem
+and make Starknet Foundry better!
+Before opening an issue it is always a good idea to search existing 
+[issues](https://github.com/foundry-rs/starknet-foundry/issues) and verify if similar one doesn't already exist. 
 
-You can also run a specific set of tests, by directly running `cargo test`.
 
-For forge tests, make sure you are in `crates/forge` directory:
+## Opening a pull request
 
-```shell
-$ cargo test --lib        # runs all unit tests
-$ cargo test integration  # runs all integration tests
-$ cargo test e2e          # runs all e2e tests
-```
-
-Similarly, to run sncast tests make sure you are in `crates/sncast` directory:
-
-```shell
-$ cargo test --lib        # runs lib unit tests
-$ cargo test helpers      # runs helpers unit tests
-$ cargo test integration  # runs all integration tests
-$ cargo test e2e          # runs all e2e tests
-```
-
-Or to run cheatnet tests make sure you are in `crates/cheatnet` directory:
+### Environment setup
```

Comment:
That's not a part of opening a pull request

Reply:
I can name this section differently, `Contributing` i think